src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -14,7 +14,9 @@ public MyHashMap() {
 
     @Override
     public void put(K key, V value) {
-        resize();
+        if (size >= DEFAULTL_CAPACITY * DEFAULT_LOAD_FACTOR) {
```

Comment:
create variable threshold and assign it value DEFAULT_CAPACITY * DEFAULT_LOAD_FACTOR
then in resize assign it a new value. Because now you only resize when threshold is 12;